src/providers/scicat-provider/dao/scicat-be-dao.ts

```diff
@@ -0,0 +1,124 @@
+import logger from "../../../server/logger";
+import { Configuration, PublishedDataApi, PublishedDataControllerFindAllRequest, PublishedDataControllerFindOneRequest } from "@scicatproject/scicat-sdk-ts-fetch";
+import { AppConfiguration } from "../../../server/app-configuration"
+
+/**
+ * This is the DAO service for Scicat. It uses a mongo connection
+ * to retrieve data.  Database connection parameters are
+ * provided by the credentials file (path defined in .env).
+ */
+export class SciCatBEConnector {
+  public static instance: SciCatBEConnector;
+  public publishedDataApi: PublishedDataApi;
+  public scicatUrl: string;
+  public publishedDataCount: number;
+  private aconf: AppConfiguration;
+
+  private constructor() {
+
+    logger.debug("Setting up scicat client.");
+
+    this.aconf = AppConfiguration.instance;
+
+    this.scicatUrl = this.aconf.scicat_backend_url;
```

Comment:
Would it throw a proper env var missing error if scicat_backend_url is not provided?

Reply:
Removed default value in configuration and added test if variable contains a string.